package.json

```diff
@@ -70,8 +70,7 @@
   "gxVersion": "0.9.0",
   "language": "go",
   "license": "",
-  "name": "floodsub",
+  "name": "go-floodsub",
```

Comment:
bonus points for renaming it to `go-libp2p-floodsub`. Matches JS (https://github.com/libp2p/js-libp2p-floodsub) and lets the users know that this is for libp2p.

Reply:
Oooo, thank you!